app/api/db/db.py

```diff
@@ -0,0 +1,88 @@
+
+from datetime import datetime, timezone
+import os
+import uuid
+from aiohttp import ClientError
+from boto3.dynamodb.conditions import Key
+import boto3
+
+class DatabaseOperations:
+    def __init__(self):
+            self.dynamodb = boto3.resource('dynamodb', region_name=os.environ.get("AWS_REGION"))
+            self.football_results = self.dynamodb.Table('bs-football-results')
+            self.football_context_prompts = self.dynamodb.Table('bs-football-context-prompts')
+            self.user_contacts = self.dynamodb.Table('bs-user-contacts')
+
+    @classmethod
+    def save_prediction(cls, address: str, prediction: str, team: str):
+        """
+        Save prediction to DynamoDB
+        """
+        if os.environ.get('ENVIRONMENT') == 'QA':
+            return { 'prediction': prediction, 'address': address, 'timestamp': timestamp}
+        timestamp = datetime.now(timezone.utc).isoformat()
+        prediction_id = str(uuid.uuid4())
+        try:
+            response = cls().football_results.query(
+                IndexName='address-team-index',
```

Comment:
I'm concerned about these hard-coding. This server is thought as a general no-input prediction service, we need to parameterized the fields we read from the DB. We have Trello cards for the Generalization of the services, we can address it later, but it is something that we need to address definitely.

Reply:
Yes, I agree. The only problem is that each event (football, comic) has its own rules, so it's difficult to generalize it